util/reggen/gen_dv.py

```diff
@@ -41,11 +65,14 @@ def miname(m: Window) -> str:
 
 def gen_core_file(outdir: str,
                   lblock: str,
-                  dv_base_prefix: str,
+                  dv_base_names: str,
```

Comment:
This should be a list, right?

Reply:
Thanks! You are faster than me :) Just updated that.